### ISSUE TITLE
Participant stuck on blank page after COV22

### ISSUE THREAD

**Issue by cusackjm** [episphere/questionnaire#163]
"After the page that asks if he feels fully recovered after having COVID, it just showed a blank white page with no buttons to push to go to a next screen."

Participant used Safari.

**Comment by danielruss** [comment]
@cusackjm The only way I can recreate the problem is my not answering the required date question SRVBLU_COV3_V1R0.  If the user took the survey with an old version of the BIOSPECIMEN (not module 1) survey (before required=true), they could have gotten that error.  I don't know when the user took the survey or ims fixed the module.

**Comment by cunnaneaq** [comment]
@danielruss @cusackjm I was able to re-create this issue in stage yesterday using Safari.  These are the screenshots of the console that I took, but can share my screen today if needed.  Connect ID: 4549730634

![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/83971268/192792721-3353b8e5-a67a-4b5a-be0e-dd5c171e3835.png)

![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/83971268/192792718-f61f3860-b63e-40e8-bb65-97a917eb3df5.png)

**Comment by cusackjm** [comment]
Tested the Participant scenario that Aileen was able to replicate and find an error, and could move past "yes mostly". @Davinkjohnson can I close this and #183 ?

**Comment by Davinkjohnson** [comment]
@cusackjm is the error still occurring? (If it is, I think we'd want to keep the issue for that error open)

**Comment by cunnaneaq** [comment]
Just tested using the same answers as last time (when I got the error/was able to replicate the issue reported) and it worked this time.  I was able to continue through the biospec mod (Stage; Safari; laptop)

**Comment by cusackjm** [comment]
@Davinkjohnson no, I was able to get through the qx without any problems. We had thought all the participant reported issues that involved them getting stuck around COV19-COV22 were all similar and likely fixed with the same fix daniel made yesterday. So was just wondering if I can close this issue and those 3 issues too

**Comment by Davinkjohnson** [comment]
Is this change on quest prod? My only thought is that we should hold these open until NORC says they see these participants have closed the blood/urine survey.

**Comment by cusackjm** [comment]
@danielruss is this is quest prod? the participant still saw an issue on 10/5

**Comment by cusackjm** [comment]
pushed to prod on 10/18, had not been updated in 29 days

**Comment by cusackjm** [comment]
From Stephanie on 11/8: Another participant (HP) reported that they are getting stuck when they enter “Yes, mostly” to the “Following your COVID-19 infection in, do you feel that you have fully recovered to your usual state of health?” question in the sample survey. They last attempted to complete the survey yesterday. We suggested that they try a different device and/or browser; they have tried using Microsoft Edge on Android and Windows 10. The Connect ID is 3342756222.

11/9: I tested as an HP participant in Stage, using Edge and Windows 10, with the participant's responses. I was unable to replicate this issue. Laura or Gene to test using their Android. @danielruss @anthonypetersen if they can replicate this issue on their Android, will we be able to right click/inspect to determine the issue?

**Comment by anthonypetersen** [comment]
Should be able to
